hub/instruction_handling/create/triggers/intro.tex

```diff
@@ -1,14 +1,16 @@
 In this section we specify the following ``module trigger shorthands.''
-\begin{multicols}{2}
+\begin{multicols}{3}
 \begin{itemize}
 	\item \locTriggerMxp{}
 	\item \locTriggerStp{}
-	\item \locTriggerOob{}
 	\item \locTriggerMmu{}
+	\item \locTriggerOob{}
+	\item \locTriggerOobX{}
+	\item \locTriggerOobU{}
```

Comment:
`trigger_OOB` now split in two (`trigger_OOB_X` and `trigger_OOB_U`) as explained earlier:
- e`X`cecptional
- `U`nexceptional